hunts/templates/tools.html

```diff
@@ -0,0 +1,63 @@
+{% extends "base.html" %}
+{% block page_content %}
+
+<div class="row justify-content-between" style="margin:10">
+    <div class="col-10">
+        <h1>Tools and References</h1>
+    </div>
+
+    <div class="col-10">
+        <h2>Word Puzzles</h2>
+    </div>
+    <div class="col-10">
+        <ul>
+            <li><a href="https://quipqiup.com/">quipqiup</a></li>
+            <li><a href="https://nutrimatic.org/">nutrimatic</a></li>
+            <li><a href="https://www.onelook.com/">OneLook</a></li>
+            <li><a href="https://www.thewordfinder.com/anagram-solver/">Anagram Solver</a></li>
+            <li><a href="https://www.highlightpress.com.au/">Cryptics Reference</a></li>
+            <li><a href="http://www.npinsker.me/puzzles/crossword">Crossword Parser</a></li>
+        </ul>
+    </div>
+
+    <div class="col-10">
+        <h2>Logic Puzzles</h2>
+    </div>
+    <div class="col-10">
+        <ul>
+            <li>Ask Max</li>
+        </ul>
+    </div>
+
+    <div class="col-10">
+        <h2>Math</h2>
+    </div>
+    <div class="col-10">
+        <ul>
+            <li><a href="https://oeis.org/">The On-Line Encyclopedia of Integer Sequences</a></li>
+            <li><a href="https://www.paulschou.com/tools/xlate/">Encodings</a></li>
+        </ul>
+    </div>
+
+    <div class="col-10">
+        <h2>Technical</h2>
```

Comment:
Solving jigsaw in GIMP: http://nastyissues.blogspot.com/2019/11/solving-jigsaw-puzzle-in-gimp.html